index.html

```diff
@@ -703,6 +703,141 @@ <h3 class="text-0 color-accent">1+1=3</h3>
       </div>
     </section>
 
+    <section id="faq" class="faq">
+      <div class="container">
+        <div>
+          <p class="text-2 text-uppercase text-center">часті запитання</p>
+        </div>
+        <div class="row d-flex d-grid justify-content-around">
```

Comment:
`d-flex` or `d-grid`? Two different `display` simultaneously?
`d-flex` by default in `row`.